cmd/affinitygroup_show.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/exoscale/cli/table"
+	"github.com/spf13/cobra"
+)
+
+// affinitygroupShowCmd represents the affinitygroup show command
+var affinitygroupShowCmd = &cobra.Command{
+	Use:     "show <name | id>",
+	Short:   "Show affinity group details",
+	Aliases: gShowAlias,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+		return showAffinityGroup(args[0])
```

Comment:
why only one?

Reply:
Because all `exo * show` commands accept only one argument.

Reply:
then it should fail on `len(args) != 1` rather than silently ignore the other ones.

Reply:
Fair enough – fixed.